src/caretogether-pwa/.env.development

```diff
@@ -4,4 +4,7 @@ VITE_APP_AUTH_CLIENT_ID=c02dcc3a-2b74-4fef-b9ef-fc1c9e83454c
 VITE_APP_AUTH_AUTHORITY=https://caretogetherb2cdev.b2clogin.com/caretogetherb2cdev.onmicrosoft.com/B2C_1A_SIGNUP_SIGNIN
 VITE_APP_AUTH_KNOWN_AUTHORITY=caretogetherb2cdev.b2clogin.com
 VITE_APP_AUTH_REDIRECT_URI=http://localhost:3000
-VITE_APP_AUTH_SCOPES=https://caretogetherb2cdev.onmicrosoft.com/cms/v1/Access
\ No newline at end of file
+VITE_APP_AUTH_SCOPES=https://caretogetherb2cdev.onmicrosoft.com/cms/v1/Access
+
+VITE_APP_PUBLIC_POSTHOG_KEY=phc_W1D3ENe9BnQS18UsldYTiDyI51tlqFIDy2p8BldZpI3

```

Comment:
Can you create a separate project in PostHog for the dev/staging environment? https://posthog.com/tutorials/multiple-environments

Reply:
done